chef-server-stats/folsom-stats

```diff
@@ -7,12 +7,15 @@
 fetch_metric({MetricKey, [MetricType = {type, histogram} ]}) ->
     {MetricKey, rpc:call(?ERCHEF, folsom_metrics,get_histogram_statistics, [MetricKey]), MetricType};
```

Comment:
I was trying to not change anything for ec11 to keep things back compat. I'm not sure executing get_metric_value will result in the same thing in ec11 as it does in cs12 but I'll try it out Monday and I guess we'll see!